base/src/main/java/com/austindroids/austinfeedsme/data/FilterableEventsRepository.kt

```diff
@@ -0,0 +1,23 @@
+package com.austindroids.austinfeedsme.data
+
+import io.reactivex.Observable
+import javax.inject.Singleton
+
+@Singleton
+open class FilterableEventsRepository(private val eventsRemoteDataSource: FilterableEventDataSource)
```

Comment:
If you use `@Inject constructor` then you won't need this in a module provides.

However if this should be single instance, and it has no `@Inject constructor`, then `@Singleton` is meaningless.